BUILD.md

```diff
@@ -72,31 +78,125 @@ below, built in LLVM, and Xcode LLVM don't support the C++20 feature set require
 API isn't support on macOS, Vulkan must be used. Additionally Vulkan must be used through the 
 Molten-VK compatibility layer
 
-### On Apple Silicon Macs, Rosetta 2 and the x86_64 version of Homebrew must be used
+The rest of this section will walk you through the process of setting up and building Cemu on a Mac, whether it's an Intel or Apple Silicon machine.
 
-You can skip this section if you have an Intel Mac. Every time you compile, you need to perform steps 2.
+### First time setup before compiling Cemu for Apple Silicon Macs
 
-1. `softwareupdate --install-rosetta` # Install Rosetta 2 if you don't have it. This only has to be done once
-2. `arch -x86_64 zsh` # run an x64 shell
+1. Install Rosetta 2.
 
-### Installing brew
+   ```bash
+   softwareupdate --install-rosetta
+   ```
 
-1. `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
-2. `eval "$(/usr/local/Homebrew/bin/brew shellenv)"` # set x86_64 brew env
+1. Run an x86_64 shell.
 
-### Installing dependencies
+   ```bash
+   arch -x86_64 zsh
+   ```
 
-`brew install boost git cmake llvm ninja nasm molten-vk`
+1. Unload your arm64-specific brew from your `PATH` so that it doesn't confuse your x86_64-specific installation of brew.
 
-### Build Cemu using cmake and clang
-1. `git clone --recursive https://github.com/cemu-project/Cemu`
-2. `cd Cemu`
-3. `cmake -S . -B build -DCMAKE_BUILD_TYPE=release -DCMAKE_C_COMPILER=/usr/local/opt/llvm/bin/clang -DCMAKE_CXX_COMPILER=/usr/local/opt/llvm/bin/clang++ -G Ninja`
-4. `cmake --build build`
-5. You should now have a Cemu executable file in the /bin folder, which you can run using `./bin/Cemu_release`.
+   ```bash
+   export PATH=`printf '%s:' $(echo $PATH | tr ':' '\n' | grep -iv "^\/opt\/homebrew\/")`
+   ```
 
-#### Troubleshooting steps
-- If step 3 gives you an error about not being able to find ninja, try appending `-DCMAKE_MAKE_PROGRAM=/usr/local/bin/ninja` to the command and running it again.
+   * **Note:** This step is only necessary if you have an arm64-specific brew installed. To check, do `echo $PATH | tr ':' '\n'`, and check if there's any mention of `/opt/homebrew`.
+1. Now continue onto the `First time setup for Intel Macs` section. Note that you should definitely do the `Install brew` step **even if** you have an arm64-specific brew already installed. This will install a second one, specific to x86_64.
+
+### First time setup before compiling Cemu for either kind of Mac
+
+1. Install brew.
+
+   ```bash
+   /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

Comment:
Using subshells like this can create issues if the downloaded script is larger than `ARG_MAX`; see also <https://porkmail.org/era/unix/award#backticks>. I believe this could be replaced by the following:

```suggestion
   curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh | /bin/bash
```

Reply:
This suggestion happens to differ from the install page of homebrew, but the article you share convinced me that it's a good idea, thanks 👍